### ISSUE TITLE
Two false positives

### ISSUE THREAD

**Issue by JordanP** [haypo/sixer#4]
Hi,
I tried sixer on OpenStack Tempest today. Awesome script ! Thanks.

It reported two false positives, i.e sixer emitted a warning but it shouldn't have.  Here is a basic reproducer.

```
import six

foo = "my print false positive"
six.next(iter())
```

And Sixer emits:

```
Warnings:
WARNING: [next] false.py: six.next(iter())
WARNING: [print] false.py: foo="my print false positive"
```

It shouldn't complain about `six.next`, the code already managed to be compatible with Py2 and Py3.

**Comment by vstinner** [comment]
Yeah, the next warning can be fixed.

The print warning cannot be easily fixed. sixer is stupid, it doesn't understand context like "statement" or "string", it only see a long string of characters. I don't plan to implement a real parser, I prefer to keep sixer simple and emit warnings. Just ignore the print warning ;-)

**Comment by vstinner** [comment]
It's now fixed in sixer 1.4. Thanks for the bug report.

By the way, you can replace six.next() with next(). I don't understand the purpose of six.next(), next() just works fine.
